.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  jvm:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        java-version:
+          - 1.8
+          - 9
+          - 10
+          # TODO:
```

Comment:
maven-compiler-plugin chokes on these. Probably best to switch to Gradle first anyway...